Jenkinsfile

```diff
@@ -1,14 +1,20 @@
 @Library('forge-shared-library')_
 
 pipeline {
+    options {
+        disableConcurrentBuilds()
```

Comment:
This shouldnt be needed, its in Forge's because gradle is dumb on large projects